<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_image"/>
    <addaction name="actionLoad_tree"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonBackground">
       <property name="text">
        <string>Background</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonImageColor">
       <property name="text">
        <string>Image color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderImageMinWindow">
       <property name="maximum">
        <number>1024</number>
       </property>
       <property name="pageStep">
        <number>128</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderImageMaxWindow">
       <property name="maximum">
        <number>1024</number>
       </property>
       <property name="pageStep">
        <number>128</number>
       </property>
       <property name="value">
        <number>1024</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderImageMinOpacity">
       <property name="maximum">
        <number>1024</number>
       </property>
       <property name="pageStep">
        <number>128</number>
       </property>
       <property name="value">
        <number>1024</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderImageMaxOpacity">
       <property name="maximum">
        <number>1024</number>
       </property>
       <property name="pageStep">
        <number>128</number>
       </property>
       <property name="value">
        <number>1024</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonTreeColor">
       <property name="text">
        <string>Tree color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderTreeOpacity">
       <property name="maximum">
        <number>1024</number>
       </property>
       <property name="pageStep">
        <number>128</number>
       </property>
       <property name="value">
        <number>1024</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionLoad_image">
   <property name="text">
    <string>Load image</string>
   </property>
  </action>
  <action name="actionLoad_tree">
   <property name="text">
    <string>Load tree</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
